src/libtriton/arch/x86/x86Specifications.cpp

```diff
@@ -50,7 +50,7 @@ namespace triton {
                                                       X86_LOWER,                                            \
                                                       true)                                                 \
                               );                                                                            \
-            name2id.emplace(#LOWER_NAME, ID_REG_X86_##UPPER_NAME);
+          name2id.emplace(#LOWER_NAME, ID_REG_X86_##UPPER_NAME);
```

Comment:
This was done to remove compile time warnings. This doesn't change the way the code works, but if this _was_ intended to be part of the above if statement, then brackets need to be added.